seq/src/seq.yml

```diff
@@ -0,0 +1,48 @@
+name: seq
+version: "0.0.0"
+author: Francis Murillo <francisavmurillo@gmail.com>
+about: "Print a sequence of numbers"
+long_about: >
+  Print numbers from FIRST to LAST, in steps of INCREMENT.
+usage: >
+
+  [OPTIONS] <LAST>
+
+  [OPTIONS] <FIRST> <LAST>
+
+  [OPTIONS] <FIRST> <INCREMENT> <LAST>
+args:
+    - FIRST:
+        help: >
```

Comment:
When possible, use single line string

Reply:
Alright, will apply that to the remaining single help values